File: examples/common_features/species_1.cpp
#include <sycomore/Species.h>
#include <sycomore/units.h>

int main()
{
    using namespace sycomore::units;
    
    // Create a Species from either relaxation times, relaxation rates or both
    sycomore::Species species_1(1000*ms, 100*ms);
    sycomore::Species species_2(1*Hz, 10*Hz);
    sycomore::Species species_3(1000*ms, 10*Hz);
    
    return 0;
}

File: examples/common_features/species_1.py
import sycomore
from sycomore.units import *

# Create a Species from either relaxation times, relaxation rates or both
species = sycomore.Species(1000*ms, 100*ms)
species = sycomore.Species(1*Hz, 10*Hz)
species = sycomore.Species(1000*ms, 10*Hz)
